conference.js

```diff
@@ -1496,38 +1492,12 @@ export default {
         }
 
         if (toggle) {
-            const wasVideoMuted = this.isLocalVideoMuted();
-
             try {
                 await this._switchToScreenSharing(options);
-            } catch (err) {
-                logger.error('Failed to switch to screensharing', err);
 
                 return;
-            }
-            if (wasVideoMuted) {
-                return;
-            }
-            const { height } = this.localVideo.track.getSettings();
-            const defaultCamera
-                = getUserSelectedCameraDeviceId(APP.store.getState());
-            let effect;
-
-            try {
-                effect = await this._createPresenterStreamEffect(
```

Comment:
What was happening here? How was effect being used?

Reply:
The effect was getting applied on the desktop stream to start the presenter mode.

Reply:
Okay. And this is part of the off-by-default change.